README.md

```diff
@@ -1,2 +1,69 @@
-# fire-data-processing
-Scripts to produce the fuel moisture and flammability data for the AFMS system
+##### Fire Data Processing Repository
+# ANU-WALD
+
+The Fire Data Processing repository is a collection of scripts used to produce the fuel moisture and flammability data for usage on the online interactive [Australian Flammability Monitoring system (AFMS)](http://wenfo.org/afms/#_/Fenner%3AFMC%3AC6/gfe_fire_weather/_/_/_/4/8/2017/_). This is a research initiative to provide quality spatial information on fire hazards in Australia to better understand the risk of bushfires, their severity and preparing ways to respond to them.
+
+
+### Authors
+---
+##### Code
+*[Zac Hatfield Dodds](https://github.com/Zac-HD), [Dion Misic](https://github.com/kingdion)*
+
+##### Science
+*[Marta Yebra](https://researchers.anu.edu.au/researchers/yebra-m)*
+
+### Important Links
+---
+[The Project](http://www.bnhcrc.com.au/research/understanding-mitigating-hazards/255)
+
+[ANU Water and Landscape Dynamics Group](http://www.wenfo.org/wald/)
+
+[Australian Flammability Monitoring System](http://wenfo.org/afms/#_/Fenner%3AFMC%3AC6/gfe_fire_weather/_/_/_/4/8/2017/_)
+
+[Australian Environment Explorer](http://wenfo.org/ausenv/#/2016/Exposed_soil/Grid/Actual/States_and%20Territories/bar,options/-23.07/135.08/4/none/Roadmap/Opaque)
+
+
+### Input Data and Process
+---
+
+### Scripts
+---
+#### onetile.py
+Onetile is used to generate *one tile* of LFMC data for the [AFMS](http://wenfo.org/afms/#_/Fenner%3AFMC%3AC6/gfe_fire_weather/_/_/_/4/8/2017/_) system. The script uses a group of colour channels within satellite meta-data from [NASA's MODIS satellite](https://terra.nasa.gov/about/terra-instruments/modis) (sinusoidal projection) to calculate information about fuel moisture and reflectance on a specific tile of the world. Onetile is also capable of interpreting SPOT6 and SPOT7 satellite data and can be extended to support more. The world map on the sinusoidal projection grid can be found [here](https://modis-land.gsfc.nasa.gov/images/MODIS_sinusoidal_grid1.gif).
+
+The script takes a few command-line arguments to generate data however is more commonly used by launchmany.py - [see below](#launchmany.py)
```

Comment:
Instead of listing them here, I'd just note that all scripts accept a `--help` argument to print usage information.